t/jobs/content-addressed.nix

```diff
@@ -0,0 +1,28 @@
+let cfg =  import ./config.nix; in
+let
+  mkDerivation = args: cfg.mkDerivation ({
+    __contentAddressed = true;
+    outputHashMode = "recursive";
+    outputHashAlgo = "sha256";
+  } // args);
```

Comment:
Feel free to extend config.nix.in (and regenerate with make) with more functions, like makeContetAddressedDerivation :)

Reply:
Right. Done in 8ef36cc7cf95cdf5b1ff7b629b9dc0f498f72548